### ISSUE TITLE
Is this still needed in django 2.x?

### ISSUE THREAD

**Issue by stardust85** [hulu/sqlite_checkforeignkeys#1]
In django 2.x I have trouble w/ sqlite fk check, but only in tests. Does this fix my problem?

**Comment by stardust85** [comment]
You may be also interested in this ticket: https://code.djangoproject.com/ticket/29928

**Comment by mckaysalisbury** [comment]
@stardust85 Yes, turning on fk for tests is exactly the reason this was built.
